installer/bundle/bundle_skel.sh

```diff
@@ -41,6 +41,27 @@ INSTALL_TYPE=<INSTALL_TYPE>
 SCRIPT_LEN=<SCRIPT_LEN>
 SCRIPT_LEN_PLUS_ONE=<SCRIPT_LEN+1>
 
+# Error codes and categories:
+
+# User configuration/parameters:
```

Comment:
Given that anyone reading these is reading our source code, they will know it's from OMS, because they're reading the bundle itself or this file in our repository. None of these strings get printed out, so I'll leave them as is.